src/tsk_fs_file.rs

```diff
@@ -450,3 +500,38 @@ impl<'fs, 'f> Iterator for TskFsAttrIterator<'fs, 'f> {
         Some(current)
     }
 }
+
+/// An iterator over TskFsFile. It reads the file in chuncks and return the chunck as vec.
+/// * ts_fs_file: is a reference to the TskFsFile that you want to read.
+/// * offset: keep track of the offset to be used for the next read operation.
+/// * file_size: the file size in bytes.
+/// * buf_size: the chunck size in bytes.
+pub struct TskFsFileIterator<'fs>{
```

Comment:
This type of iteration is not needed with Read/Seek implementation

Reply:
This is has been removed